src/server/nuclearnet/fake_nuclearnet_client.ts

```diff
@@ -29,8 +36,9 @@ export class FakeNUClearNetClient implements NUClearNetClient {
   public connect(options: NUClearNetOptions): () => void {
     this.peer = {
       name: options.name,
-      address: '127.0.0.1',
-      port: 7447,
+      // Address and port are not used for anything yet, dummy values used.
+      address: 'fake_address',
```

Comment:
haha indeed! The idea is that any code that tries to will break since its not designed to be used.